### ISSUE TITLE
Signature error on download

### ISSUE THREAD

**Issue by Wilfred** [jacksonrayhamilton/tern-context-coloring#4]
When trying to install from MELPA, I get:

```
Failed to verify signature context-coloring-8.0.1.tar.sig:
Bad signature from 474F05837FBDEF9B GNU ELPA Signing Agent <elpasign@elpa.gnu.org>
Command output:
gpg: Signature made Tue 31 May 2016 05:05:02 EDT
gpg:                using DSA key 474F05837FBDEF9B
gpg: BAD signature from "GNU ELPA Signing Agent <elpasign@elpa.gnu.org>" [unknown]
```

Is this expected? Am I getting corrupt data, or does this package require me to add a key to my keyring?

**Comment by jacksonrayhamilton** [comment]
How are you installing the package? I just tried to install it via `list-packages` and didn't get an error like that.

**Comment by Wilfred** [comment]
Just from `list-packages`: I pressed `i` to mark the package, then `x` to install it.

**Comment by jacksonrayhamilton** [comment]
``` sh
emacs --version # GNU Emacs 25.1.1
emacs -Q -eval '(progn (require '"'"'package) (add-to-list '"'"'package-archives '"'"'("melpa" . "https://melpa.org/packages/")) (package-initialize))'
```

Yep, still no errors for me. And after the install, in `~/.emacs.d/elpa/context-coloring-8.0.1.signed`, I see:

```
Good signature from 474F05837FBDEF9B GNU ELPA Signing Agent <elpasign@elpa.gnu.org> (trust undefined) created at 2016-05-31T02:05:02-0700 using DSA
```

**Comment by Wilfred** [comment]
I can no longer reproduce this issue, so I'm going to speculate that I had data corruption. Sorry for the noise.

**Comment by indrajithi** [comment]
when I run `package-refresh-contents` in emacs I get the same error:
```
Failed to verify signature archive-contents.sig:                                                                                                                                                                    
2Bad signature from 474F05837FBDEF9B GNU ELPA Signing Agent <elpasign@elpa.gnu.org>                                                                                                                                  
3Command output:                                                                                                                                                                                                     
4gpg: Signature made Tuesday 02 April 2019 02:40:04 PM IST                                                                                                                                                           
5gpg:                using DSA key CA442C00F91774F17F59D9B0474F05837FBDEF9B                                                                                                                                          
6gpg: BAD signature from "GNU ELPA Signing Agent <elpasign@elpa.gnu.org>" [unknown]
```
emacs version: 26.1

**Comment by jacksonrayhamilton** [comment]
Hi @indrajithi,

Sorry, I don’t think I can provide much assistance.  Like last time, I tried installing via `list-packages` and still didn’t have any issues — in fact, I got a “Good signature.”

Maybe the GNU ELPA server was hiccuping.  I suggest you try again, maybe with a fresh GPG configuration.  Otherwise, report a bug to Emacs: https://www.gnu.org/software/emacs/manual/html_node/efaq/Reporting-bugs.html
